zcash_client_backend/src/data_api/wallet.rs

```diff
@@ -1,15 +1,15 @@
 use std::convert::Infallible;
 use std::fmt::Debug;
 
+use incrementalmerkletree::MerklePath;
 use zcash_primitives::{
     consensus::{self, NetworkUpgrade},
     memo::MemoBytes,
-    merkle_tree::MerklePath,
```

Comment:
Looking over the changes after my review, this actually confuses me, because this type still exists.